mk/run.mk

```diff
@@ -1,11 +1,9 @@
-GO_RUN := go run $(GOFLAGS) $(LD_FLAGS)
```

Comment:
Why do we want to get rid of this? just to simplify? Or did `GOFLAGS` / `LD_FLAGS` break something?

Reply:
Because the places we do `go run` are not meant to be external. For example we use it in make generate or make docs. There's no need there to have CGO_ENABLED=0 or hide build flags. This makes things simpler and quicker.